security-oauth2/src/main/java/io/micronaut/security/oauth2/endpoint/token/response/TokenErrorResponse.java

```diff
@@ -30,15 +29,16 @@
  * @since 1.2.0
  */
 @Introspected
-@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Comment:
same here

Reply:
I have reverted that change here: https://github.com/micronaut-projects/micronaut-security/pull/1395/commits/c5637c6e9e7222e3edfa1dafcf1b5bd3c1ec0b65